### DISCUSSION TITLE
Github's development repository?

### DISCUSSION THREAD

**Discussion by Noitarud** [community/community#64311] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I tried to find GitHub's development repository (because I was unsatisfied with a change and wonder if they could put a UI version setting in settings) and was directed to here which bears warnings they want discussions only. Can I have a link for getting there?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by WilliamHCarter** [comment]
Hi, unfortunately Github doesn't have a public repository for their UI, it is developed privately and therefore doesn't allow direct PRs. However, you can certainly provide that feedback here and hopefully see it considered by the development team.

**Providing Feedback**
The easiest way to propose a change and provide it as feedback would be via this discussions page, simply start a discussion under general, or maybe profile depending on the specifics of your change. Then, mark that discussion as `Product Feedback`, which should so the trick.

Hope this helps!

_if this answer was satisfactory, feel free to 'mark as answered' in the feed to help other users with similar questions_

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Noitarud ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by naserykarim922** [comment]
مدیریت وب سایتم باهوش مصنوعی. میشه انجام بدم
